tests/test_utils/test_helpers.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import torch
+from compressed_tensors import load_compressed, save_compressed
+from compressed_tensors.config import BitmaskConfig
+
+
+@pytest.fixture
+def tensors():
+    tensors = {"tensor_1": torch.Tensor([[0.0, 0.0, 0.0], [1.0, 1.0, 1.0]])}
+    return tensors
+
+
+def test_save_compressed_sparse_bitmask(tmp_path, tensors):
+    save_compressed(
+        tensors,
+        compression_format="sparse-bitmask",
+        save_path=tmp_path / "model.safetensors",
+    )
+    assert (tmp_path / "model.safetensors").exists()
```

Comment:
Could we check the compression config too?

Reply:
But we are not creating a compression config here. It would need to be created separately for this atomic function.